data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/threshold/VoltageThreshold.java

```diff
@@ -41,4 +45,24 @@ public double getMaxValue() {
     public void setMaxValue(double maxValue) {
         this.maxValue = maxValue;
     }
+
+    @Override
+    public boolean isMinThresholdOvercome(Network network, Cnec cnec) throws SynchronizationException {
+        return false;
+    }
+
+    @Override
+    public boolean isMaxThresholdOvercome(Network network, Cnec cnec) throws SynchronizationException {
+        return false;
+    }
```

Comment:
Is this finished or TODO?

Reply:
This is TODO, I replaced by NotImplementedException.